webapp/certification/api.py

```diff
@@ -0,0 +1,182 @@
+from requests import Session
+
+
+class CertificationAPI:
+    """
+    Method names and properties to describe and map directly
+    onto the Certification API
+    (at the time of writing, this API is available at
+    https://certification.canonical.com/api/v1)
+    """
+
+    def __init__(self, base_url: str, session: Session):
+        self.base_url = base_url
+        self.session = session
+
+    def _get(self, path: str, params: dict = {}):
+        # Remove "None" values from params
+        params = {
+            key: value for key, value in params.items() if value is not None
+        }
+
+        # Get the JSON data
+        response = self.session.get(
+            f"{self.base_url}/{path.strip('/')}/?format=json", params=params
+        )
+
+        # Raise any HTTP errors
+        response.raise_for_status()
+
+        return response
+
+    def certifiedmakes(
```

Comment:
@nottrobin any reason you chose not to use snake case for the methods names in this class? If not can we? I feel like it's a bit more readable.

Reply:
My guess is that the Api has properties called this `certifiedmakes`, `certifiedmodeldevices`, `componentsummaries` so we followed this naming. But I guess we can change our methods. Updated